src/components/views/elements/EventTilePreview.tsx

```diff
@@ -107,7 +107,7 @@ export default class EventTilePreview extends React.Component<IProps, IState> {
             userId: this.props.userId,
             getAvatarUrl: (..._) => {
                 return Avatar.avatarUrlForUser(
-                    { avatarUrl: this.props.avatarUrl },
+                    {avatarUrl: this.props.avatarUrl},
```

Comment:
surprised the linter is okay with this given our rule

Reply:
We only have that linter in the js-sdk ATM as its not compatible with React aiui but good point, not sure what happened here.